src/garage/torch/algos/vpg.py

```diff
@@ -61,16 +61,18 @@ def __init__(
             baseline,
             optimizer=torch.optim.Adam,
             policy_lr=_Default(1e-2),
-            max_path_length=500,
-            num_train_per_epoch=1,
-            discount=0.99,
-            gae_lambda=1,
-            center_adv=True,
-            positive_adv=False,
-            policy_ent_coeff=0.0,
-            use_softplus_entropy=False,
-            stop_entropy_gradient=False,
-            entropy_method='no_entropy',
+            max_path_length=_Default(500),
+            num_train_per_epoch=_Default(1),
+            discount=_Default(0.99),
+            gae_lambda=_Default(1),
+            center_adv=_Default(True),
+            positive_adv=_Default(False),
```

Comment:
what is the purpose of _Default?

Reply:
please see the commit which introduces the module for background. I think @krzentner left a comment about these as well.

Reply:
I still think they're unnecessary though in this case.